Plugins/Data/README.md

```diff
@@ -2,3 +2,10 @@
 @ingroup data 
 
 Provides a number of data structures for NWN code to use (simulated arrays)
+
+# Notes
+
+As of 8193.16, the Array provider has been re-written using the internal SQLite database instead of via the NWNX plugin.
+
+Two files are provided:  `inc_array` which includes a re-implementation of the data structure into SQLite, and `nwnx_data`, which is a shim to provide backwards compabitility for people still using NWNX_Array to provide time to migrate to the new version.  Included in the NWScript folder is an example implementation showing usage of Data_Array in comparison with NWNX_Data_Array.
```

Comment:
compabitility = `compatibility`

Reply:
Sorry about the nits! This is a welcome change and I think the way you've done it is right with leaving the folder structure in tact and removing the cpp. You may want to edit your initial commit to show it will close two bugs by adding
```
Closes #976 
Closes #733 
```

Reply:
I'm not familiar with editing commit messages..  Is there a guide somewhere?

Reply:
Just at the top of this github page, click the three dots in the right hand side in your "Comments welcome." comment and change that to add the lines mentioned.